utils/patches/computeAngle/src/test_cart.cpp

```diff
@@ -0,0 +1,45 @@
+#include "cart.h"
+
+#include <iostream>
+
+#include <cmath>
+
+int count = 1;
+
```

Comment:
good potential of using a std::vector here for a,b,c values - can atleast just have it for dummy values of RSSI if you feel

Reply:
No problem in having it but we will face a lot of conversion issues.

Reply:
Hmm, I just plan to have a hard-coded vector with 5 values as of now, if you can manage that. Let me know if possible

As this is just to envision our use of the same with live updates of values